code/src/App.js

```diff
@@ -1,9 +1,114 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary'
+
+const adventureGroup = ["surfing", "climbing", "hiking"];
+const destinationGroup = ["Costa Rica", "Mount Everest", "Machu Picchu"];
+const companionGroups = ["alone", "with family members", "with friends"];
 
 export const App = () => {
+  const [name, setName] = useState("");
+  const [adventures, setAdventures] = useState();
+  const [destinations, setDestinations] = useState();
+  const [mostImportant, setMostImportant] = useState("");
+  const [companionGroup, setCompanionGroup] = useState()
+  const [submitted, setSubmitted] = useState(false)
+
+
   return (
     <div>
-      Find me in src/app.js!
+
+      {!submitted && (
+
+        <form onSubmit={event => event.preventDefault()}>
+
+          <div className="heroHeader">
+            <h2>Plan your 2020 adventure!</h2>
```

Comment:
```suggestion
            <h1>Plan your 2020 adventure!</h1>
```

Reply:
I started out with H1 but it looked huge on my laptop, but maybe its the best choice when it comes to accessibility?